applications/luci-app-opkg/htdocs/luci-static/resources/view/opkg.js

```diff
@@ -294,8 +294,8 @@ function display(pattern)
 		currentDisplayRows.push([
 			name,
 			ver,
-			pkg.size ? '%.1024mB'.format(pkg.size)
-			         : (altsize ? '~%.1024mB'.format(altsize) : '-'),
+			pkg.size ? '%1024mB'.format(pkg.size)
+			         : (altsize ? '~%1024mB'.format(altsize) : '-'),
```

Comment:
Should it not be `MiB`? Also would it not be better then to convert to Mb instead?